app/script/components/mentionSuggestions.js

```diff
@@ -117,24 +107,25 @@ z.components.MentionSuggestions = class MentionSuggestions {
   }
 
   updateScrollPosition(selectedNumber) {
-    const suggestionList = document.querySelector('.mention-suggestion-list');
-    if (!suggestionList) {
+    const scrollContentSelector = '.conversation-input-bar-mention-suggestion > .simplebar-scroll-content';
```

Comment:
Let's try not to depend on classes we don't manage ourselves. Here we create an implicit dependency to `simplebar`.
There might be some cleaner way to get the scroll height